2023-10-04 16:52:43,755 INFO    MainThread:15492 [internal.py:wandb_internal():90] W&B internal server running at pid: 15492, started at: 2023-10-04 16:52:43.755002
2023-10-04 16:52:43,759 INFO    WriterThread:15492 [datastore.py:open_for_write():75] open: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/run-3s8t6wta.wandb
2023-10-04 16:52:43,762 DEBUG   SenderThread:15492 [sender.py:send():236] send: header
2023-10-04 16:52:43,763 DEBUG   SenderThread:15492 [sender.py:send():236] send: run
2023-10-04 16:52:44,368 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: check_version
2023-10-04 16:52:44,373 INFO    SenderThread:15492 [dir_watcher.py:__init__():216] watching files in: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files
2023-10-04 16:52:44,373 INFO    SenderThread:15492 [sender.py:_start_run_threads():815] run started: 3s8t6wta with start time 1696409562
2023-10-04 16:52:44,373 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:52:44,373 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:52:44,374 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: check_version
2023-10-04 16:52:44,736 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: run_start
2023-10-04 16:52:44,751 DEBUG   HandlerThread:15492 [meta.py:__init__():35] meta init
2023-10-04 16:52:44,751 DEBUG   HandlerThread:15492 [meta.py:__init__():49] meta init done
2023-10-04 16:52:44,751 DEBUG   HandlerThread:15492 [meta.py:probe():209] probe
2023-10-04 16:52:44,759 DEBUG   HandlerThread:15492 [meta.py:_setup_git():199] setup git
2023-10-04 16:52:44,785 DEBUG   HandlerThread:15492 [meta.py:_setup_git():206] setup git done
2023-10-04 16:52:44,786 DEBUG   HandlerThread:15492 [meta.py:_save_pip():53] save pip
2023-10-04 16:52:44,786 DEBUG   HandlerThread:15492 [meta.py:_save_pip():67] save pip done
2023-10-04 16:52:44,786 DEBUG   HandlerThread:15492 [meta.py:_save_conda():74] save conda
2023-10-04 16:52:45,375 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_created():269] file/dir created: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/conda-environment.yaml
2023-10-04 16:52:45,376 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_created():269] file/dir created: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/requirements.txt
2023-10-04 16:52:45,376 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_created():269] file/dir created: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/wandb-summary.json
2023-10-04 16:52:50,376 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_created():269] file/dir created: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/output.log
2023-10-04 16:52:50,657 DEBUG   HandlerThread:15492 [meta.py:_save_conda():84] save conda done
2023-10-04 16:52:50,657 DEBUG   HandlerThread:15492 [meta.py:probe():247] probe done
2023-10-04 16:52:50,661 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: stop_status
2023-10-04 16:52:50,662 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: stop_status
2023-10-04 16:52:51,105 DEBUG   SenderThread:15492 [sender.py:send():236] send: telemetry
2023-10-04 16:52:51,106 DEBUG   SenderThread:15492 [sender.py:send():236] send: files
2023-10-04 16:52:51,106 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-metadata.json with policy now
2023-10-04 16:52:51,376 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/conda-environment.yaml
2023-10-04 16:52:51,377 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_created():269] file/dir created: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/wandb-metadata.json
2023-10-04 16:52:51,598 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/wandb-metadata.json?Expires=1696495971&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=tLaEylTIQpB7KVF9D70OVIhTi8rAna5RRNsv%2B39iuLCzjmZUtEznhvwhIarunfBlV8aP9ockJDf3pceuEnvJlThK6bljxy9AKM4uVXbp2ZCljl2FGCrQFi5uAhd9%2BRvVrceUfQkGtK%2B1%2BZGLW8x%2FuNsL5OKpg6rgsj%2FsbD%2BpMg8cI7wF%2FpEmrhYyohKy9XC1Nl3DpQenXe27HAQVLxbHG87SPJNcU98vE5EWqAPHSvM2FJJ59IbJ3gquWE%2FQpIenBxMfL4berqCyfI3zYvvW%2B%2BjA576JZxzYX4wE3y6TSoN45BHUXdNGF829cdiqOQ00zihOYHGYsv5tsCyO9xYk6A%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:52:51,599 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '830'}
2023-10-04 16:52:51,599 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:52:52,867 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/wandb-metadata.json?Expires=1696495971&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=tLaEylTIQpB7KVF9D70OVIhTi8rAna5RRNsv%2B39iuLCzjmZUtEznhvwhIarunfBlV8aP9ockJDf3pceuEnvJlThK6bljxy9AKM4uVXbp2ZCljl2FGCrQFi5uAhd9%2BRvVrceUfQkGtK%2B1%2BZGLW8x%2FuNsL5OKpg6rgsj%2FsbD%2BpMg8cI7wF%2FpEmrhYyohKy9XC1Nl3DpQenXe27HAQVLxbHG87SPJNcU98vE5EWqAPHSvM2FJJ59IbJ3gquWE%2FQpIenBxMfL4berqCyfI3zYvvW%2B%2BjA576JZxzYX4wE3y6TSoN45BHUXdNGF829cdiqOQ00zihOYHGYsv5tsCyO9xYk6A%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:52:52,867 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '830'}
2023-10-04 16:52:52,867 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:52:55,146 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/wandb-metadata.json?Expires=1696495971&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=tLaEylTIQpB7KVF9D70OVIhTi8rAna5RRNsv%2B39iuLCzjmZUtEznhvwhIarunfBlV8aP9ockJDf3pceuEnvJlThK6bljxy9AKM4uVXbp2ZCljl2FGCrQFi5uAhd9%2BRvVrceUfQkGtK%2B1%2BZGLW8x%2FuNsL5OKpg6rgsj%2FsbD%2BpMg8cI7wF%2FpEmrhYyohKy9XC1Nl3DpQenXe27HAQVLxbHG87SPJNcU98vE5EWqAPHSvM2FJJ59IbJ3gquWE%2FQpIenBxMfL4berqCyfI3zYvvW%2B%2BjA576JZxzYX4wE3y6TSoN45BHUXdNGF829cdiqOQ00zihOYHGYsv5tsCyO9xYk6A%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:52:55,147 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '830'}
2023-10-04 16:52:55,147 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:52:55,147 ERROR   Thread-11 :15492 [retry.py:__call__():132] Retry attempt failed:
Traceback (most recent call last):
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/urllib3/connectionpool.py", line 1042, in _validate_conn
    conn.connect()
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/urllib3/connection.py", line 414, in connect
    self.sock = ssl_wrap_socket(
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 449, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 493, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
ConnectionResetError: [Errno 104] Connection reset by peer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/requests/adapters.py", line 489, in send
    resp = conn.urlopen(
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/urllib3/connectionpool.py", line 787, in urlopen
    retries = retries.increment(
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/urllib3/util/retry.py", line 550, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/urllib3/packages/six.py", line 769, in reraise
    raise value.with_traceback(tb)
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/urllib3/connectionpool.py", line 1042, in _validate_conn
    conn.connect()
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/urllib3/connection.py", line 414, in connect
    self.sock = ssl_wrap_socket(
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 449, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 493, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 1691, in upload_file
    response = requests.put(url, data=progress, headers=extra_headers)
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/requests/api.py", line 130, in put
    return request("put", url, data=data, **kwargs)
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/requests/sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/requests/sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/requests/adapters.py", line 547, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/wandb/sdk/lib/retry.py", line 108, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 1707, in upload_file
    raise e.with_traceback(sys.exc_info()[2])
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 1691, in upload_file
    response = requests.put(url, data=progress, headers=extra_headers)
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/requests/api.py", line 130, in put
    return request("put", url, data=data, **kwargs)
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/requests/sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/requests/sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "/home/youyx/anaconda3/envs/pose2mesh-env/lib/python3.8/site-packages/requests/adapters.py", line 547, in send
    raise ConnectionError(err, request=request)
wandb.sdk.lib.retry.TransientError: None
2023-10-04 16:52:58,380 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/output.log
2023-10-04 16:53:00,163 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/wandb-metadata.json?Expires=1696495971&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=tLaEylTIQpB7KVF9D70OVIhTi8rAna5RRNsv%2B39iuLCzjmZUtEznhvwhIarunfBlV8aP9ockJDf3pceuEnvJlThK6bljxy9AKM4uVXbp2ZCljl2FGCrQFi5uAhd9%2BRvVrceUfQkGtK%2B1%2BZGLW8x%2FuNsL5OKpg6rgsj%2FsbD%2BpMg8cI7wF%2FpEmrhYyohKy9XC1Nl3DpQenXe27HAQVLxbHG87SPJNcU98vE5EWqAPHSvM2FJJ59IbJ3gquWE%2FQpIenBxMfL4berqCyfI3zYvvW%2B%2BjA576JZxzYX4wE3y6TSoN45BHUXdNGF829cdiqOQ00zihOYHGYsv5tsCyO9xYk6A%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:00,163 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '830'}
2023-10-04 16:53:00,163 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:00,381 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/output.log
2023-10-04 16:53:08,637 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/wandb-metadata.json?Expires=1696495971&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=tLaEylTIQpB7KVF9D70OVIhTi8rAna5RRNsv%2B39iuLCzjmZUtEznhvwhIarunfBlV8aP9ockJDf3pceuEnvJlThK6bljxy9AKM4uVXbp2ZCljl2FGCrQFi5uAhd9%2BRvVrceUfQkGtK%2B1%2BZGLW8x%2FuNsL5OKpg6rgsj%2FsbD%2BpMg8cI7wF%2FpEmrhYyohKy9XC1Nl3DpQenXe27HAQVLxbHG87SPJNcU98vE5EWqAPHSvM2FJJ59IbJ3gquWE%2FQpIenBxMfL4berqCyfI3zYvvW%2B%2BjA576JZxzYX4wE3y6TSoN45BHUXdNGF829cdiqOQ00zihOYHGYsv5tsCyO9xYk6A%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:08,637 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '830'}
2023-10-04 16:53:08,637 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:15,389 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/config.yaml
2023-10-04 16:53:16,948 DEBUG   SenderThread:15492 [sender.py:send():236] send: stats
2023-10-04 16:53:27,298 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: stop_status
2023-10-04 16:53:27,299 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: stop_status
2023-10-04 16:53:27,971 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/wandb-metadata.json?Expires=1696495971&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=tLaEylTIQpB7KVF9D70OVIhTi8rAna5RRNsv%2B39iuLCzjmZUtEznhvwhIarunfBlV8aP9ockJDf3pceuEnvJlThK6bljxy9AKM4uVXbp2ZCljl2FGCrQFi5uAhd9%2BRvVrceUfQkGtK%2B1%2BZGLW8x%2FuNsL5OKpg6rgsj%2FsbD%2BpMg8cI7wF%2FpEmrhYyohKy9XC1Nl3DpQenXe27HAQVLxbHG87SPJNcU98vE5EWqAPHSvM2FJJ59IbJ3gquWE%2FQpIenBxMfL4berqCyfI3zYvvW%2B%2BjA576JZxzYX4wE3y6TSoN45BHUXdNGF829cdiqOQ00zihOYHGYsv5tsCyO9xYk6A%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:27,971 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '830'}
2023-10-04 16:53:27,971 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:37,400 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/output.log
2023-10-04 16:53:38,052 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:38,053 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:38,054 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:38,054 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:38,308 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:38,309 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:38,310 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:38,310 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:38,400 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/wandb-summary.json
2023-10-04 16:53:38,538 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:38,539 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:38,539 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:38,540 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:38,745 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:38,747 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:38,748 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:38,748 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:38,926 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:38,928 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:38,928 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:38,929 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:39,145 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:39,146 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:39,147 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:39,147 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:39,339 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:39,340 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:39,341 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:39,341 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:39,401 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/wandb-summary.json
2023-10-04 16:53:39,401 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/output.log
2023-10-04 16:53:39,566 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:39,568 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:39,569 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:39,569 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:39,755 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:39,757 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:39,757 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:39,758 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:39,974 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:39,974 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:39,975 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:39,975 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:40,177 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:40,179 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:40,179 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:40,180 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:40,372 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:40,372 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:40,373 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:40,373 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:40,402 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/wandb-summary.json
2023-10-04 16:53:40,578 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:40,579 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:40,580 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:40,581 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:40,798 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:40,799 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:40,799 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:40,800 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:41,015 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:41,016 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:41,016 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:41,017 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:41,193 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:41,195 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:41,195 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:41,196 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:41,402 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/wandb-summary.json
2023-10-04 16:53:41,403 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/output.log
2023-10-04 16:53:41,450 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:41,451 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:41,452 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:41,452 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:41,685 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:41,686 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:41,687 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:41,688 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:41,901 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:41,902 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:41,903 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:41,904 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:42,110 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:42,112 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:42,112 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:42,113 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:42,281 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:42,282 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:42,283 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:42,284 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:42,403 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/wandb-summary.json
2023-10-04 16:53:42,500 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:42,501 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:42,501 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:42,502 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:42,694 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:42,695 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:42,695 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:42,696 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:42,699 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: stop_status
2023-10-04 16:53:42,699 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: stop_status
2023-10-04 16:53:42,868 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:43,089 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:43,122 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:43,123 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:43,124 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:43,124 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:43,125 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:43,125 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:43,276 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:43,277 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:43,277 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:43,278 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:43,403 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/wandb-summary.json
2023-10-04 16:53:43,404 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/output.log
2023-10-04 16:53:43,454 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: partial_history
2023-10-04 16:53:43,455 DEBUG   SenderThread:15492 [sender.py:send():236] send: history
2023-10-04 16:53:43,456 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:43,457 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:43,532 WARNING MainThread:15492 [internal.py:wandb_internal():153] Internal process interrupt: 1
2023-10-04 16:53:43,727 DEBUG   SenderThread:15492 [sender.py:send():236] send: telemetry
2023-10-04 16:53:44,404 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/wandb-summary.json
2023-10-04 16:53:45,404 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/output.log
2023-10-04 16:53:45,953 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:45,953 DEBUG   SenderThread:15492 [sender.py:send():236] send: exit
2023-10-04 16:53:45,954 INFO    SenderThread:15492 [sender.py:send_exit():372] handling exit code: 255
2023-10-04 16:53:45,954 INFO    SenderThread:15492 [sender.py:send_exit():374] handling runtime: 61
2023-10-04 16:53:45,955 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:45,955 INFO    SenderThread:15492 [sender.py:send_exit():380] send defer
2023-10-04 16:53:45,956 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:45,956 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: defer
2023-10-04 16:53:45,957 INFO    HandlerThread:15492 [handler.py:handle_request_defer():163] handle defer: 0
2023-10-04 16:53:45,957 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: defer
2023-10-04 16:53:45,957 INFO    SenderThread:15492 [sender.py:send_request_defer():389] handle sender defer: 0
2023-10-04 16:53:45,957 INFO    SenderThread:15492 [sender.py:transition_state():393] send defer: 1
2023-10-04 16:53:45,958 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: defer
2023-10-04 16:53:45,958 INFO    HandlerThread:15492 [handler.py:handle_request_defer():163] handle defer: 1
2023-10-04 16:53:46,214 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: defer
2023-10-04 16:53:46,215 INFO    SenderThread:15492 [sender.py:send_request_defer():389] handle sender defer: 1
2023-10-04 16:53:46,215 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:46,215 INFO    SenderThread:15492 [sender.py:transition_state():393] send defer: 2
2023-10-04 16:53:46,216 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:46,216 DEBUG   SenderThread:15492 [sender.py:send():236] send: stats
2023-10-04 16:53:46,217 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: defer
2023-10-04 16:53:46,218 INFO    HandlerThread:15492 [handler.py:handle_request_defer():163] handle defer: 2
2023-10-04 16:53:46,218 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: defer
2023-10-04 16:53:46,218 INFO    SenderThread:15492 [sender.py:send_request_defer():389] handle sender defer: 2
2023-10-04 16:53:46,218 INFO    SenderThread:15492 [sender.py:transition_state():393] send defer: 3
2023-10-04 16:53:46,219 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: defer
2023-10-04 16:53:46,219 INFO    HandlerThread:15492 [handler.py:handle_request_defer():163] handle defer: 3
2023-10-04 16:53:46,219 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: defer
2023-10-04 16:53:46,220 INFO    SenderThread:15492 [sender.py:send_request_defer():389] handle sender defer: 3
2023-10-04 16:53:46,220 INFO    SenderThread:15492 [sender.py:transition_state():393] send defer: 4
2023-10-04 16:53:46,220 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: defer
2023-10-04 16:53:46,220 INFO    HandlerThread:15492 [handler.py:handle_request_defer():163] handle defer: 4
2023-10-04 16:53:46,220 DEBUG   SenderThread:15492 [sender.py:send():236] send: summary
2023-10-04 16:53:46,221 INFO    SenderThread:15492 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-10-04 16:53:46,222 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: defer
2023-10-04 16:53:46,222 INFO    SenderThread:15492 [sender.py:send_request_defer():389] handle sender defer: 4
2023-10-04 16:53:46,222 INFO    SenderThread:15492 [sender.py:transition_state():393] send defer: 5
2023-10-04 16:53:46,222 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: defer
2023-10-04 16:53:46,222 INFO    HandlerThread:15492 [handler.py:handle_request_defer():163] handle defer: 5
2023-10-04 16:53:46,223 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: defer
2023-10-04 16:53:46,223 INFO    SenderThread:15492 [sender.py:send_request_defer():389] handle sender defer: 5
2023-10-04 16:53:46,223 INFO    SenderThread:15492 [sender.py:transition_state():393] send defer: 6
2023-10-04 16:53:46,223 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: defer
2023-10-04 16:53:46,223 INFO    HandlerThread:15492 [handler.py:handle_request_defer():163] handle defer: 6
2023-10-04 16:53:46,224 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: defer
2023-10-04 16:53:46,224 INFO    SenderThread:15492 [sender.py:send_request_defer():389] handle sender defer: 6
2023-10-04 16:53:46,224 INFO    SenderThread:15492 [dir_watcher.py:finish():353] shutting down directory watcher
2023-10-04 16:53:46,319 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:46,405 INFO    Thread-7  :15492 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/wandb-summary.json
2023-10-04 16:53:46,406 INFO    SenderThread:15492 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/config.yaml
2023-10-04 16:53:46,406 INFO    SenderThread:15492 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/output.log
2023-10-04 16:53:46,407 INFO    SenderThread:15492 [dir_watcher.py:finish():383] scan: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files
2023-10-04 16:53:46,407 INFO    SenderThread:15492 [dir_watcher.py:finish():397] scan save: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/wandb-summary.json wandb-summary.json
2023-10-04 16:53:46,407 INFO    SenderThread:15492 [dir_watcher.py:finish():397] scan save: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/requirements.txt requirements.txt
2023-10-04 16:53:46,408 INFO    SenderThread:15492 [dir_watcher.py:finish():397] scan save: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/wandb-metadata.json wandb-metadata.json
2023-10-04 16:53:46,413 INFO    SenderThread:15492 [dir_watcher.py:finish():397] scan save: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/config.yaml config.yaml
2023-10-04 16:53:46,414 INFO    SenderThread:15492 [dir_watcher.py:finish():397] scan save: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/output.log output.log
2023-10-04 16:53:46,419 INFO    SenderThread:15492 [dir_watcher.py:finish():397] scan save: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/conda-environment.yaml conda-environment.yaml
2023-10-04 16:53:46,427 INFO    SenderThread:15492 [sender.py:transition_state():393] send defer: 7
2023-10-04 16:53:46,432 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:46,436 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: defer
2023-10-04 16:53:46,437 INFO    HandlerThread:15492 [handler.py:handle_request_defer():163] handle defer: 7
2023-10-04 16:53:46,437 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: defer
2023-10-04 16:53:46,438 INFO    SenderThread:15492 [sender.py:send_request_defer():389] handle sender defer: 7
2023-10-04 16:53:46,438 INFO    SenderThread:15492 [file_pusher.py:finish():171] shutting down file pusher
2023-10-04 16:53:46,537 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:46,538 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:46,640 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:46,640 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:46,743 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:46,743 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:46,845 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:46,846 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:46,912 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/requirements.txt?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Q4xKRWSk7uGEVAmog%2BNCxXLarlYLyvYWpP9XlF6d95eDlcix63PZPWgCu7IN2kW6WFD8D0R%2BW6P%2FgHKf6rBW1zJpWzjuw1xYUFAR%2FxuZ07lwQEDIyMLUh5wUn2703wf%2BlhRnOkox9%2FRXlLlnuvIz8p5bFoqfE3ms%2BjiUt4DIimA2F5OGbroUuIYCxKnVvlSfAgZC0CB8i7sJdDMO3PahbifD9rQ4HG52xsHKaODJIM8EPCTXYWvE%2FzrQW%2BMCvkYiwUsjE999%2FO9qNP4zdoGxfssfDgnzuiXmR8OGeKdOkNkDNjjZD8kpWOgbzpe6uvNBDbjBlnK1uwWbmFJqZ9crRw%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:46,912 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2067'}
2023-10-04 16:53:46,912 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:46,948 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:46,948 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:46,949 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/wandb-summary.json?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IKd2QT561gAHYhANHPmgx0MlRG%2FmSVoskmxBB71nAtzCdVM2Pa1NhftkJFT0maBTvl%2BFm3hWenXhoBUq4LVE8nOFsSB17ekwUrhj1M9SzkmSvAH28WQXKL%2FbuxgHLu%2B9xT0dO5myycK3DmiUF7jBtlv1UKmYFVu8rr1KXnzWmxgcwiAbSjX7eS07q6w5PF4FBZoDpNNqfArZefAWNEhaCSDzCTZ79I8mrFZyKKIfDPb5LppoP4vnOquLxOO4kkT1iPcCLl2Q2RsOGmHa8J8YmDEM%2Fyh93tym4k6p7KlIP3XLf8MgoNVCwnNwAxkIomxceXobcoDB858tuGbnZENGoQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:46,949 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '359'}
2023-10-04 16:53:46,950 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:46,977 ERROR   Thread-16 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/conda-environment.yaml?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DIxreMJgP4qDahDF1L9HIOcbAwYrwhvIHD3MprFN%2FHzoK9EKZNOM%2BjDDquUuhJDIK3g7mp903O3AIn5RGWSEUUsyT%2B9tEEyTIxFUFauYhO%2BL%2FSgc%2B693YnEelD5%2Bt8Sx9KrBrFNsCmQQhqW%2BBzzkpaqzUQ%2FP1%2BeY61Y3lntEk0T8VC0gD2UFxcybE%2B2BFM1%2FI1%2BvChmjUhd2aO3NfkoFktDRgA5YC8BCaDRFbQnx69ETesKxCYd9R0jxaVI1p%2FNTDy1ePBUGaKVJRaRJLeURVQsN6IFePhieoYini7h3sg%2FM%2FcnA75JL7iOMD%2Bnz5R7cnTzKhEMdL%2FxGWgPYMChO2g%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:46,977 ERROR   Thread-16 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '4790'}
2023-10-04 16:53:46,977 ERROR   Thread-16 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:46,985 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/output.log?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=abdFuryOpbdwyc2kVfnen1GFQDp7KadJ%2BBpIMnPGpT0Qll9ZTa4LzymQFUfeZ4yN4Kfswa8WXaMGHj%2BRCjH6xb1PKQNGSZTKKGRoPQkGclXhFSbXsO64fIf5I2oduwjBU1heRpBz2mvCy3qSLgBiTFMCIT%2BFdCnzufuDbQg4twHKKOdAVH1BmZ4HBraumuBkEm76KPTWj0ZL6huhiWsu6oLlZSgsR%2FTVXtGFy9tLnrbIpibC7va2zpti7Mfpwygzn%2FlT1K5u%2F9qSZjecPxYiARrB1iqD9o3Xh4YWK7uAfbcpc7QRp1bnChngHl%2BWYBh5Ed%2BSKIuca%2FS3Nl%2Bj0s0ofQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:46,985 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1053'}
2023-10-04 16:53:46,986 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:46,989 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/config.yaml?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Jnq322JRnliPFcjtgHJ5FJ5W7O1YmRQsJqW1Wv9Hunk7gjKpnz1L%2FBU6yIiVYFp5MFImlKY0lkCxD1BILzYm4%2Fo0ltBwYRrEFzPUD4zjkLq5%2BVAtw6c4TK7AtgFupRNVruJ3ws1luH9EoAZzU%2B%2F5Uxm3oZ22aKHc9vamXUlVxYSRudrNHcgXfOneIdWt0blUQclYg1Wlll%2Ff5g8Nr2NSJ35%2BjpDgv1Vvs8zS7SZOb%2Fwg0t8qyl3FSMojOLuSUpgYHKL0z36spehYND9P0Jr6XEy9MLLY8upmaKLUxC99%2Bm7ehYPzgIc5LZbfYGEtuwqi4cDBi0OYppO3aDtzd83g9A%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:46,989 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2405'}
2023-10-04 16:53:46,989 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:47,051 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:47,052 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:47,154 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:47,155 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:47,257 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:47,258 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:47,360 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:47,361 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:47,463 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:47,464 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:47,566 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:47,566 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:47,669 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:47,669 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:47,772 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:47,772 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:47,875 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:47,875 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:47,977 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:47,978 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:48,013 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/requirements.txt?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Q4xKRWSk7uGEVAmog%2BNCxXLarlYLyvYWpP9XlF6d95eDlcix63PZPWgCu7IN2kW6WFD8D0R%2BW6P%2FgHKf6rBW1zJpWzjuw1xYUFAR%2FxuZ07lwQEDIyMLUh5wUn2703wf%2BlhRnOkox9%2FRXlLlnuvIz8p5bFoqfE3ms%2BjiUt4DIimA2F5OGbroUuIYCxKnVvlSfAgZC0CB8i7sJdDMO3PahbifD9rQ4HG52xsHKaODJIM8EPCTXYWvE%2FzrQW%2BMCvkYiwUsjE999%2FO9qNP4zdoGxfssfDgnzuiXmR8OGeKdOkNkDNjjZD8kpWOgbzpe6uvNBDbjBlnK1uwWbmFJqZ9crRw%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:48,013 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2067'}
2023-10-04 16:53:48,013 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:48,080 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:48,081 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:48,141 ERROR   Thread-16 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/conda-environment.yaml?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DIxreMJgP4qDahDF1L9HIOcbAwYrwhvIHD3MprFN%2FHzoK9EKZNOM%2BjDDquUuhJDIK3g7mp903O3AIn5RGWSEUUsyT%2B9tEEyTIxFUFauYhO%2BL%2FSgc%2B693YnEelD5%2Bt8Sx9KrBrFNsCmQQhqW%2BBzzkpaqzUQ%2FP1%2BeY61Y3lntEk0T8VC0gD2UFxcybE%2B2BFM1%2FI1%2BvChmjUhd2aO3NfkoFktDRgA5YC8BCaDRFbQnx69ETesKxCYd9R0jxaVI1p%2FNTDy1ePBUGaKVJRaRJLeURVQsN6IFePhieoYini7h3sg%2FM%2FcnA75JL7iOMD%2Bnz5R7cnTzKhEMdL%2FxGWgPYMChO2g%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:48,141 ERROR   Thread-16 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '4790'}
2023-10-04 16:53:48,141 ERROR   Thread-16 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:48,183 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:48,183 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:48,211 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/output.log?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=abdFuryOpbdwyc2kVfnen1GFQDp7KadJ%2BBpIMnPGpT0Qll9ZTa4LzymQFUfeZ4yN4Kfswa8WXaMGHj%2BRCjH6xb1PKQNGSZTKKGRoPQkGclXhFSbXsO64fIf5I2oduwjBU1heRpBz2mvCy3qSLgBiTFMCIT%2BFdCnzufuDbQg4twHKKOdAVH1BmZ4HBraumuBkEm76KPTWj0ZL6huhiWsu6oLlZSgsR%2FTVXtGFy9tLnrbIpibC7va2zpti7Mfpwygzn%2FlT1K5u%2F9qSZjecPxYiARrB1iqD9o3Xh4YWK7uAfbcpc7QRp1bnChngHl%2BWYBh5Ed%2BSKIuca%2FS3Nl%2Bj0s0ofQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:48,211 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1053'}
2023-10-04 16:53:48,211 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:48,211 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/config.yaml?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Jnq322JRnliPFcjtgHJ5FJ5W7O1YmRQsJqW1Wv9Hunk7gjKpnz1L%2FBU6yIiVYFp5MFImlKY0lkCxD1BILzYm4%2Fo0ltBwYRrEFzPUD4zjkLq5%2BVAtw6c4TK7AtgFupRNVruJ3ws1luH9EoAZzU%2B%2F5Uxm3oZ22aKHc9vamXUlVxYSRudrNHcgXfOneIdWt0blUQclYg1Wlll%2Ff5g8Nr2NSJ35%2BjpDgv1Vvs8zS7SZOb%2Fwg0t8qyl3FSMojOLuSUpgYHKL0z36spehYND9P0Jr6XEy9MLLY8upmaKLUxC99%2Bm7ehYPzgIc5LZbfYGEtuwqi4cDBi0OYppO3aDtzd83g9A%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:48,212 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2405'}
2023-10-04 16:53:48,212 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:48,276 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/wandb-summary.json?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IKd2QT561gAHYhANHPmgx0MlRG%2FmSVoskmxBB71nAtzCdVM2Pa1NhftkJFT0maBTvl%2BFm3hWenXhoBUq4LVE8nOFsSB17ekwUrhj1M9SzkmSvAH28WQXKL%2FbuxgHLu%2B9xT0dO5myycK3DmiUF7jBtlv1UKmYFVu8rr1KXnzWmxgcwiAbSjX7eS07q6w5PF4FBZoDpNNqfArZefAWNEhaCSDzCTZ79I8mrFZyKKIfDPb5LppoP4vnOquLxOO4kkT1iPcCLl2Q2RsOGmHa8J8YmDEM%2Fyh93tym4k6p7KlIP3XLf8MgoNVCwnNwAxkIomxceXobcoDB858tuGbnZENGoQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:48,276 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '359'}
2023-10-04 16:53:48,277 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:48,286 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:48,286 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:48,388 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:48,388 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:48,491 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:48,491 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:48,593 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:48,593 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:48,696 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:48,696 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:48,799 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:48,799 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:48,901 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:48,902 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:49,004 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:49,005 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:49,107 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:49,107 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:49,209 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:49,210 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:49,312 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:49,313 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:49,415 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:49,416 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:49,518 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:49,518 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:49,621 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:49,621 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:49,724 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:49,724 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:49,827 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:49,827 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:49,929 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:49,930 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:50,032 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:50,032 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:50,135 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:50,135 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:50,189 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/requirements.txt?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Q4xKRWSk7uGEVAmog%2BNCxXLarlYLyvYWpP9XlF6d95eDlcix63PZPWgCu7IN2kW6WFD8D0R%2BW6P%2FgHKf6rBW1zJpWzjuw1xYUFAR%2FxuZ07lwQEDIyMLUh5wUn2703wf%2BlhRnOkox9%2FRXlLlnuvIz8p5bFoqfE3ms%2BjiUt4DIimA2F5OGbroUuIYCxKnVvlSfAgZC0CB8i7sJdDMO3PahbifD9rQ4HG52xsHKaODJIM8EPCTXYWvE%2FzrQW%2BMCvkYiwUsjE999%2FO9qNP4zdoGxfssfDgnzuiXmR8OGeKdOkNkDNjjZD8kpWOgbzpe6uvNBDbjBlnK1uwWbmFJqZ9crRw%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:50,189 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2067'}
2023-10-04 16:53:50,190 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:50,238 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:50,238 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:50,340 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:50,341 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:50,443 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:50,443 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:50,534 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/config.yaml?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Jnq322JRnliPFcjtgHJ5FJ5W7O1YmRQsJqW1Wv9Hunk7gjKpnz1L%2FBU6yIiVYFp5MFImlKY0lkCxD1BILzYm4%2Fo0ltBwYRrEFzPUD4zjkLq5%2BVAtw6c4TK7AtgFupRNVruJ3ws1luH9EoAZzU%2B%2F5Uxm3oZ22aKHc9vamXUlVxYSRudrNHcgXfOneIdWt0blUQclYg1Wlll%2Ff5g8Nr2NSJ35%2BjpDgv1Vvs8zS7SZOb%2Fwg0t8qyl3FSMojOLuSUpgYHKL0z36spehYND9P0Jr6XEy9MLLY8upmaKLUxC99%2Bm7ehYPzgIc5LZbfYGEtuwqi4cDBi0OYppO3aDtzd83g9A%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:50,535 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2405'}
2023-10-04 16:53:50,535 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:50,546 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:50,546 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:50,648 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:50,648 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:50,682 ERROR   Thread-16 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/conda-environment.yaml?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DIxreMJgP4qDahDF1L9HIOcbAwYrwhvIHD3MprFN%2FHzoK9EKZNOM%2BjDDquUuhJDIK3g7mp903O3AIn5RGWSEUUsyT%2B9tEEyTIxFUFauYhO%2BL%2FSgc%2B693YnEelD5%2Bt8Sx9KrBrFNsCmQQhqW%2BBzzkpaqzUQ%2FP1%2BeY61Y3lntEk0T8VC0gD2UFxcybE%2B2BFM1%2FI1%2BvChmjUhd2aO3NfkoFktDRgA5YC8BCaDRFbQnx69ETesKxCYd9R0jxaVI1p%2FNTDy1ePBUGaKVJRaRJLeURVQsN6IFePhieoYini7h3sg%2FM%2FcnA75JL7iOMD%2Bnz5R7cnTzKhEMdL%2FxGWgPYMChO2g%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:50,682 ERROR   Thread-16 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '4790'}
2023-10-04 16:53:50,682 ERROR   Thread-16 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:50,720 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/wandb-summary.json?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IKd2QT561gAHYhANHPmgx0MlRG%2FmSVoskmxBB71nAtzCdVM2Pa1NhftkJFT0maBTvl%2BFm3hWenXhoBUq4LVE8nOFsSB17ekwUrhj1M9SzkmSvAH28WQXKL%2FbuxgHLu%2B9xT0dO5myycK3DmiUF7jBtlv1UKmYFVu8rr1KXnzWmxgcwiAbSjX7eS07q6w5PF4FBZoDpNNqfArZefAWNEhaCSDzCTZ79I8mrFZyKKIfDPb5LppoP4vnOquLxOO4kkT1iPcCLl2Q2RsOGmHa8J8YmDEM%2Fyh93tym4k6p7KlIP3XLf8MgoNVCwnNwAxkIomxceXobcoDB858tuGbnZENGoQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:50,721 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '359'}
2023-10-04 16:53:50,721 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:50,727 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/output.log?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=abdFuryOpbdwyc2kVfnen1GFQDp7KadJ%2BBpIMnPGpT0Qll9ZTa4LzymQFUfeZ4yN4Kfswa8WXaMGHj%2BRCjH6xb1PKQNGSZTKKGRoPQkGclXhFSbXsO64fIf5I2oduwjBU1heRpBz2mvCy3qSLgBiTFMCIT%2BFdCnzufuDbQg4twHKKOdAVH1BmZ4HBraumuBkEm76KPTWj0ZL6huhiWsu6oLlZSgsR%2FTVXtGFy9tLnrbIpibC7va2zpti7Mfpwygzn%2FlT1K5u%2F9qSZjecPxYiARrB1iqD9o3Xh4YWK7uAfbcpc7QRp1bnChngHl%2BWYBh5Ed%2BSKIuca%2FS3Nl%2Bj0s0ofQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:50,727 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1053'}
2023-10-04 16:53:50,727 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:50,750 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:50,751 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:50,853 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:50,854 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:50,956 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:50,956 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:51,059 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:51,059 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:51,162 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:51,162 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:51,264 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:51,265 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:51,367 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:51,368 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:51,470 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:51,470 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:51,573 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:51,573 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:51,676 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:51,676 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:51,779 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:51,779 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:51,881 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:51,881 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:51,984 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:51,984 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:52,086 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:52,086 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:52,188 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:52,189 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:52,291 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:52,291 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:52,393 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:52,394 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:52,496 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:52,496 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:52,598 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:52,599 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:52,701 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:52,701 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:52,803 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:52,803 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:52,905 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:52,906 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:53,008 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:53,008 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:53,110 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:53,111 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:53,213 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:53,213 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:53,315 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:53,316 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:53,417 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:53,418 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:53,520 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:53,520 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:53,623 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:53,623 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:53,726 DEBUG   HandlerThread:15492 [handler.py:handle_request():140] handle_request: poll_exit
2023-10-04 16:53:53,726 DEBUG   SenderThread:15492 [sender.py:send_request():250] send_request: poll_exit
2023-10-04 16:53:53,781 WARNING MainThread:15492 [internal.py:wandb_internal():153] Internal process interrupt: 2
2023-10-04 16:53:53,782 ERROR   MainThread:15492 [internal.py:wandb_internal():156] Internal process interrupted.
2023-10-04 16:53:54,222 INFO    WriterThread:15492 [datastore.py:close():279] close: /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/run-3s8t6wta.wandb
2023-10-04 16:53:54,354 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/requirements.txt?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Q4xKRWSk7uGEVAmog%2BNCxXLarlYLyvYWpP9XlF6d95eDlcix63PZPWgCu7IN2kW6WFD8D0R%2BW6P%2FgHKf6rBW1zJpWzjuw1xYUFAR%2FxuZ07lwQEDIyMLUh5wUn2703wf%2BlhRnOkox9%2FRXlLlnuvIz8p5bFoqfE3ms%2BjiUt4DIimA2F5OGbroUuIYCxKnVvlSfAgZC0CB8i7sJdDMO3PahbifD9rQ4HG52xsHKaODJIM8EPCTXYWvE%2FzrQW%2BMCvkYiwUsjE999%2FO9qNP4zdoGxfssfDgnzuiXmR8OGeKdOkNkDNjjZD8kpWOgbzpe6uvNBDbjBlnK1uwWbmFJqZ9crRw%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:54,355 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2067'}
2023-10-04 16:53:54,355 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:54,727 INFO    SenderThread:15492 [sender.py:finish():1114] shutting down sender
2023-10-04 16:53:54,727 INFO    SenderThread:15492 [file_pusher.py:finish():171] shutting down file pusher
2023-10-04 16:53:54,727 INFO    SenderThread:15492 [file_pusher.py:join():176] waiting for file pusher
2023-10-04 16:53:54,727 INFO    HandlerThread:15492 [handler.py:finish():806] shutting down handler
2023-10-04 16:53:55,245 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/wandb-summary.json?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IKd2QT561gAHYhANHPmgx0MlRG%2FmSVoskmxBB71nAtzCdVM2Pa1NhftkJFT0maBTvl%2BFm3hWenXhoBUq4LVE8nOFsSB17ekwUrhj1M9SzkmSvAH28WQXKL%2FbuxgHLu%2B9xT0dO5myycK3DmiUF7jBtlv1UKmYFVu8rr1KXnzWmxgcwiAbSjX7eS07q6w5PF4FBZoDpNNqfArZefAWNEhaCSDzCTZ79I8mrFZyKKIfDPb5LppoP4vnOquLxOO4kkT1iPcCLl2Q2RsOGmHa8J8YmDEM%2Fyh93tym4k6p7KlIP3XLf8MgoNVCwnNwAxkIomxceXobcoDB858tuGbnZENGoQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:55,246 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '359'}
2023-10-04 16:53:55,246 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:55,490 ERROR   Thread-16 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/conda-environment.yaml?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DIxreMJgP4qDahDF1L9HIOcbAwYrwhvIHD3MprFN%2FHzoK9EKZNOM%2BjDDquUuhJDIK3g7mp903O3AIn5RGWSEUUsyT%2B9tEEyTIxFUFauYhO%2BL%2FSgc%2B693YnEelD5%2Bt8Sx9KrBrFNsCmQQhqW%2BBzzkpaqzUQ%2FP1%2BeY61Y3lntEk0T8VC0gD2UFxcybE%2B2BFM1%2FI1%2BvChmjUhd2aO3NfkoFktDRgA5YC8BCaDRFbQnx69ETesKxCYd9R0jxaVI1p%2FNTDy1ePBUGaKVJRaRJLeURVQsN6IFePhieoYini7h3sg%2FM%2FcnA75JL7iOMD%2Bnz5R7cnTzKhEMdL%2FxGWgPYMChO2g%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:55,491 ERROR   Thread-16 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '4790'}
2023-10-04 16:53:55,491 ERROR   Thread-16 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:55,604 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/config.yaml?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Jnq322JRnliPFcjtgHJ5FJ5W7O1YmRQsJqW1Wv9Hunk7gjKpnz1L%2FBU6yIiVYFp5MFImlKY0lkCxD1BILzYm4%2Fo0ltBwYRrEFzPUD4zjkLq5%2BVAtw6c4TK7AtgFupRNVruJ3ws1luH9EoAZzU%2B%2F5Uxm3oZ22aKHc9vamXUlVxYSRudrNHcgXfOneIdWt0blUQclYg1Wlll%2Ff5g8Nr2NSJ35%2BjpDgv1Vvs8zS7SZOb%2Fwg0t8qyl3FSMojOLuSUpgYHKL0z36spehYND9P0Jr6XEy9MLLY8upmaKLUxC99%2Bm7ehYPzgIc5LZbfYGEtuwqi4cDBi0OYppO3aDtzd83g9A%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:55,604 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2405'}
2023-10-04 16:53:55,604 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:53:55,686 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/output.log?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=abdFuryOpbdwyc2kVfnen1GFQDp7KadJ%2BBpIMnPGpT0Qll9ZTa4LzymQFUfeZ4yN4Kfswa8WXaMGHj%2BRCjH6xb1PKQNGSZTKKGRoPQkGclXhFSbXsO64fIf5I2oduwjBU1heRpBz2mvCy3qSLgBiTFMCIT%2BFdCnzufuDbQg4twHKKOdAVH1BmZ4HBraumuBkEm76KPTWj0ZL6huhiWsu6oLlZSgsR%2FTVXtGFy9tLnrbIpibC7va2zpti7Mfpwygzn%2FlT1K5u%2F9qSZjecPxYiARrB1iqD9o3Xh4YWK7uAfbcpc7QRp1bnChngHl%2BWYBh5Ed%2BSKIuca%2FS3Nl%2Bj0s0ofQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:53:55,686 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1053'}
2023-10-04 16:53:55,686 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:54:03,190 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/requirements.txt?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Q4xKRWSk7uGEVAmog%2BNCxXLarlYLyvYWpP9XlF6d95eDlcix63PZPWgCu7IN2kW6WFD8D0R%2BW6P%2FgHKf6rBW1zJpWzjuw1xYUFAR%2FxuZ07lwQEDIyMLUh5wUn2703wf%2BlhRnOkox9%2FRXlLlnuvIz8p5bFoqfE3ms%2BjiUt4DIimA2F5OGbroUuIYCxKnVvlSfAgZC0CB8i7sJdDMO3PahbifD9rQ4HG52xsHKaODJIM8EPCTXYWvE%2FzrQW%2BMCvkYiwUsjE999%2FO9qNP4zdoGxfssfDgnzuiXmR8OGeKdOkNkDNjjZD8kpWOgbzpe6uvNBDbjBlnK1uwWbmFJqZ9crRw%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:54:03,191 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2067'}
2023-10-04 16:54:03,191 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:54:03,955 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/wandb-summary.json?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IKd2QT561gAHYhANHPmgx0MlRG%2FmSVoskmxBB71nAtzCdVM2Pa1NhftkJFT0maBTvl%2BFm3hWenXhoBUq4LVE8nOFsSB17ekwUrhj1M9SzkmSvAH28WQXKL%2FbuxgHLu%2B9xT0dO5myycK3DmiUF7jBtlv1UKmYFVu8rr1KXnzWmxgcwiAbSjX7eS07q6w5PF4FBZoDpNNqfArZefAWNEhaCSDzCTZ79I8mrFZyKKIfDPb5LppoP4vnOquLxOO4kkT1iPcCLl2Q2RsOGmHa8J8YmDEM%2Fyh93tym4k6p7KlIP3XLf8MgoNVCwnNwAxkIomxceXobcoDB858tuGbnZENGoQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:54:03,956 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '359'}
2023-10-04 16:54:03,956 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:54:04,284 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/output.log?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=abdFuryOpbdwyc2kVfnen1GFQDp7KadJ%2BBpIMnPGpT0Qll9ZTa4LzymQFUfeZ4yN4Kfswa8WXaMGHj%2BRCjH6xb1PKQNGSZTKKGRoPQkGclXhFSbXsO64fIf5I2oduwjBU1heRpBz2mvCy3qSLgBiTFMCIT%2BFdCnzufuDbQg4twHKKOdAVH1BmZ4HBraumuBkEm76KPTWj0ZL6huhiWsu6oLlZSgsR%2FTVXtGFy9tLnrbIpibC7va2zpti7Mfpwygzn%2FlT1K5u%2F9qSZjecPxYiARrB1iqD9o3Xh4YWK7uAfbcpc7QRp1bnChngHl%2BWYBh5Ed%2BSKIuca%2FS3Nl%2Bj0s0ofQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:54:04,284 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1053'}
2023-10-04 16:54:04,284 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:54:04,352 INFO    Thread-16 :15492 [upload_job.py:push():138] Uploaded file /home/youyx/models/PMCE/lib/core/../../experiment/exp_10-04_16_52/wandb/run-20231004_165242-3s8t6wta/files/conda-environment.yaml
2023-10-04 16:54:04,381 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/config.yaml?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Jnq322JRnliPFcjtgHJ5FJ5W7O1YmRQsJqW1Wv9Hunk7gjKpnz1L%2FBU6yIiVYFp5MFImlKY0lkCxD1BILzYm4%2Fo0ltBwYRrEFzPUD4zjkLq5%2BVAtw6c4TK7AtgFupRNVruJ3ws1luH9EoAZzU%2B%2F5Uxm3oZ22aKHc9vamXUlVxYSRudrNHcgXfOneIdWt0blUQclYg1Wlll%2Ff5g8Nr2NSJ35%2BjpDgv1Vvs8zS7SZOb%2Fwg0t8qyl3FSMojOLuSUpgYHKL0z36spehYND9P0Jr6XEy9MLLY8upmaKLUxC99%2Bm7ehYPzgIc5LZbfYGEtuwqi4cDBi0OYppO3aDtzd83g9A%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:54:04,381 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2405'}
2023-10-04 16:54:04,382 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:54:08,011 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/wandb-metadata.json?Expires=1696495971&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=tLaEylTIQpB7KVF9D70OVIhTi8rAna5RRNsv%2B39iuLCzjmZUtEznhvwhIarunfBlV8aP9ockJDf3pceuEnvJlThK6bljxy9AKM4uVXbp2ZCljl2FGCrQFi5uAhd9%2BRvVrceUfQkGtK%2B1%2BZGLW8x%2FuNsL5OKpg6rgsj%2FsbD%2BpMg8cI7wF%2FpEmrhYyohKy9XC1Nl3DpQenXe27HAQVLxbHG87SPJNcU98vE5EWqAPHSvM2FJJ59IbJ3gquWE%2FQpIenBxMfL4berqCyfI3zYvvW%2B%2BjA576JZxzYX4wE3y6TSoN45BHUXdNGF829cdiqOQ00zihOYHGYsv5tsCyO9xYk6A%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:54:08,011 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '830'}
2023-10-04 16:54:08,011 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:54:19,790 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/requirements.txt?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Q4xKRWSk7uGEVAmog%2BNCxXLarlYLyvYWpP9XlF6d95eDlcix63PZPWgCu7IN2kW6WFD8D0R%2BW6P%2FgHKf6rBW1zJpWzjuw1xYUFAR%2FxuZ07lwQEDIyMLUh5wUn2703wf%2BlhRnOkox9%2FRXlLlnuvIz8p5bFoqfE3ms%2BjiUt4DIimA2F5OGbroUuIYCxKnVvlSfAgZC0CB8i7sJdDMO3PahbifD9rQ4HG52xsHKaODJIM8EPCTXYWvE%2FzrQW%2BMCvkYiwUsjE999%2FO9qNP4zdoGxfssfDgnzuiXmR8OGeKdOkNkDNjjZD8kpWOgbzpe6uvNBDbjBlnK1uwWbmFJqZ9crRw%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:54:19,790 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2067'}
2023-10-04 16:54:19,790 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:54:20,877 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/wandb-summary.json?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IKd2QT561gAHYhANHPmgx0MlRG%2FmSVoskmxBB71nAtzCdVM2Pa1NhftkJFT0maBTvl%2BFm3hWenXhoBUq4LVE8nOFsSB17ekwUrhj1M9SzkmSvAH28WQXKL%2FbuxgHLu%2B9xT0dO5myycK3DmiUF7jBtlv1UKmYFVu8rr1KXnzWmxgcwiAbSjX7eS07q6w5PF4FBZoDpNNqfArZefAWNEhaCSDzCTZ79I8mrFZyKKIfDPb5LppoP4vnOquLxOO4kkT1iPcCLl2Q2RsOGmHa8J8YmDEM%2Fyh93tym4k6p7KlIP3XLf8MgoNVCwnNwAxkIomxceXobcoDB858tuGbnZENGoQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:54:20,877 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '359'}
2023-10-04 16:54:20,877 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:54:21,121 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/output.log?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=abdFuryOpbdwyc2kVfnen1GFQDp7KadJ%2BBpIMnPGpT0Qll9ZTa4LzymQFUfeZ4yN4Kfswa8WXaMGHj%2BRCjH6xb1PKQNGSZTKKGRoPQkGclXhFSbXsO64fIf5I2oduwjBU1heRpBz2mvCy3qSLgBiTFMCIT%2BFdCnzufuDbQg4twHKKOdAVH1BmZ4HBraumuBkEm76KPTWj0ZL6huhiWsu6oLlZSgsR%2FTVXtGFy9tLnrbIpibC7va2zpti7Mfpwygzn%2FlT1K5u%2F9qSZjecPxYiARrB1iqD9o3Xh4YWK7uAfbcpc7QRp1bnChngHl%2BWYBh5Ed%2BSKIuca%2FS3Nl%2Bj0s0ofQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:54:21,121 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1053'}
2023-10-04 16:54:21,121 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:54:22,748 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/config.yaml?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Jnq322JRnliPFcjtgHJ5FJ5W7O1YmRQsJqW1Wv9Hunk7gjKpnz1L%2FBU6yIiVYFp5MFImlKY0lkCxD1BILzYm4%2Fo0ltBwYRrEFzPUD4zjkLq5%2BVAtw6c4TK7AtgFupRNVruJ3ws1luH9EoAZzU%2B%2F5Uxm3oZ22aKHc9vamXUlVxYSRudrNHcgXfOneIdWt0blUQclYg1Wlll%2Ff5g8Nr2NSJ35%2BjpDgv1Vvs8zS7SZOb%2Fwg0t8qyl3FSMojOLuSUpgYHKL0z36spehYND9P0Jr6XEy9MLLY8upmaKLUxC99%2Bm7ehYPzgIc5LZbfYGEtuwqi4cDBi0OYppO3aDtzd83g9A%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:54:22,748 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2405'}
2023-10-04 16:54:22,748 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:54:54,790 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/requirements.txt?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Q4xKRWSk7uGEVAmog%2BNCxXLarlYLyvYWpP9XlF6d95eDlcix63PZPWgCu7IN2kW6WFD8D0R%2BW6P%2FgHKf6rBW1zJpWzjuw1xYUFAR%2FxuZ07lwQEDIyMLUh5wUn2703wf%2BlhRnOkox9%2FRXlLlnuvIz8p5bFoqfE3ms%2BjiUt4DIimA2F5OGbroUuIYCxKnVvlSfAgZC0CB8i7sJdDMO3PahbifD9rQ4HG52xsHKaODJIM8EPCTXYWvE%2FzrQW%2BMCvkYiwUsjE999%2FO9qNP4zdoGxfssfDgnzuiXmR8OGeKdOkNkDNjjZD8kpWOgbzpe6uvNBDbjBlnK1uwWbmFJqZ9crRw%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:54:54,790 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2067'}
2023-10-04 16:54:54,790 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:54:55,744 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/output.log?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=abdFuryOpbdwyc2kVfnen1GFQDp7KadJ%2BBpIMnPGpT0Qll9ZTa4LzymQFUfeZ4yN4Kfswa8WXaMGHj%2BRCjH6xb1PKQNGSZTKKGRoPQkGclXhFSbXsO64fIf5I2oduwjBU1heRpBz2mvCy3qSLgBiTFMCIT%2BFdCnzufuDbQg4twHKKOdAVH1BmZ4HBraumuBkEm76KPTWj0ZL6huhiWsu6oLlZSgsR%2FTVXtGFy9tLnrbIpibC7va2zpti7Mfpwygzn%2FlT1K5u%2F9qSZjecPxYiARrB1iqD9o3Xh4YWK7uAfbcpc7QRp1bnChngHl%2BWYBh5Ed%2BSKIuca%2FS3Nl%2Bj0s0ofQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:54:55,744 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1053'}
2023-10-04 16:54:55,744 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:54:57,736 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/wandb-summary.json?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IKd2QT561gAHYhANHPmgx0MlRG%2FmSVoskmxBB71nAtzCdVM2Pa1NhftkJFT0maBTvl%2BFm3hWenXhoBUq4LVE8nOFsSB17ekwUrhj1M9SzkmSvAH28WQXKL%2FbuxgHLu%2B9xT0dO5myycK3DmiUF7jBtlv1UKmYFVu8rr1KXnzWmxgcwiAbSjX7eS07q6w5PF4FBZoDpNNqfArZefAWNEhaCSDzCTZ79I8mrFZyKKIfDPb5LppoP4vnOquLxOO4kkT1iPcCLl2Q2RsOGmHa8J8YmDEM%2Fyh93tym4k6p7KlIP3XLf8MgoNVCwnNwAxkIomxceXobcoDB858tuGbnZENGoQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:54:57,736 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '359'}
2023-10-04 16:54:57,736 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:54:59,141 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/config.yaml?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Jnq322JRnliPFcjtgHJ5FJ5W7O1YmRQsJqW1Wv9Hunk7gjKpnz1L%2FBU6yIiVYFp5MFImlKY0lkCxD1BILzYm4%2Fo0ltBwYRrEFzPUD4zjkLq5%2BVAtw6c4TK7AtgFupRNVruJ3ws1luH9EoAZzU%2B%2F5Uxm3oZ22aKHc9vamXUlVxYSRudrNHcgXfOneIdWt0blUQclYg1Wlll%2Ff5g8Nr2NSJ35%2BjpDgv1Vvs8zS7SZOb%2Fwg0t8qyl3FSMojOLuSUpgYHKL0z36spehYND9P0Jr6XEy9MLLY8upmaKLUxC99%2Bm7ehYPzgIc5LZbfYGEtuwqi4cDBi0OYppO3aDtzd83g9A%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:54:59,141 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2405'}
2023-10-04 16:54:59,141 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:55:17,483 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/wandb-metadata.json?Expires=1696495971&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=tLaEylTIQpB7KVF9D70OVIhTi8rAna5RRNsv%2B39iuLCzjmZUtEznhvwhIarunfBlV8aP9ockJDf3pceuEnvJlThK6bljxy9AKM4uVXbp2ZCljl2FGCrQFi5uAhd9%2BRvVrceUfQkGtK%2B1%2BZGLW8x%2FuNsL5OKpg6rgsj%2FsbD%2BpMg8cI7wF%2FpEmrhYyohKy9XC1Nl3DpQenXe27HAQVLxbHG87SPJNcU98vE5EWqAPHSvM2FJJ59IbJ3gquWE%2FQpIenBxMfL4berqCyfI3zYvvW%2B%2BjA576JZxzYX4wE3y6TSoN45BHUXdNGF829cdiqOQ00zihOYHGYsv5tsCyO9xYk6A%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:55:17,483 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '830'}
2023-10-04 16:55:17,483 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:55:59,993 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/output.log?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=abdFuryOpbdwyc2kVfnen1GFQDp7KadJ%2BBpIMnPGpT0Qll9ZTa4LzymQFUfeZ4yN4Kfswa8WXaMGHj%2BRCjH6xb1PKQNGSZTKKGRoPQkGclXhFSbXsO64fIf5I2oduwjBU1heRpBz2mvCy3qSLgBiTFMCIT%2BFdCnzufuDbQg4twHKKOdAVH1BmZ4HBraumuBkEm76KPTWj0ZL6huhiWsu6oLlZSgsR%2FTVXtGFy9tLnrbIpibC7va2zpti7Mfpwygzn%2FlT1K5u%2F9qSZjecPxYiARrB1iqD9o3Xh4YWK7uAfbcpc7QRp1bnChngHl%2BWYBh5Ed%2BSKIuca%2FS3Nl%2Bj0s0ofQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:55:59,994 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1053'}
2023-10-04 16:55:59,994 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:56:06,101 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/config.yaml?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Jnq322JRnliPFcjtgHJ5FJ5W7O1YmRQsJqW1Wv9Hunk7gjKpnz1L%2FBU6yIiVYFp5MFImlKY0lkCxD1BILzYm4%2Fo0ltBwYRrEFzPUD4zjkLq5%2BVAtw6c4TK7AtgFupRNVruJ3ws1luH9EoAZzU%2B%2F5Uxm3oZ22aKHc9vamXUlVxYSRudrNHcgXfOneIdWt0blUQclYg1Wlll%2Ff5g8Nr2NSJ35%2BjpDgv1Vvs8zS7SZOb%2Fwg0t8qyl3FSMojOLuSUpgYHKL0z36spehYND9P0Jr6XEy9MLLY8upmaKLUxC99%2Bm7ehYPzgIc5LZbfYGEtuwqi4cDBi0OYppO3aDtzd83g9A%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:56:06,102 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2405'}
2023-10-04 16:56:06,102 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:56:09,361 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/requirements.txt?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Q4xKRWSk7uGEVAmog%2BNCxXLarlYLyvYWpP9XlF6d95eDlcix63PZPWgCu7IN2kW6WFD8D0R%2BW6P%2FgHKf6rBW1zJpWzjuw1xYUFAR%2FxuZ07lwQEDIyMLUh5wUn2703wf%2BlhRnOkox9%2FRXlLlnuvIz8p5bFoqfE3ms%2BjiUt4DIimA2F5OGbroUuIYCxKnVvlSfAgZC0CB8i7sJdDMO3PahbifD9rQ4HG52xsHKaODJIM8EPCTXYWvE%2FzrQW%2BMCvkYiwUsjE999%2FO9qNP4zdoGxfssfDgnzuiXmR8OGeKdOkNkDNjjZD8kpWOgbzpe6uvNBDbjBlnK1uwWbmFJqZ9crRw%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:56:09,362 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2067'}
2023-10-04 16:56:09,362 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:56:15,422 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/wandb-summary.json?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IKd2QT561gAHYhANHPmgx0MlRG%2FmSVoskmxBB71nAtzCdVM2Pa1NhftkJFT0maBTvl%2BFm3hWenXhoBUq4LVE8nOFsSB17ekwUrhj1M9SzkmSvAH28WQXKL%2FbuxgHLu%2B9xT0dO5myycK3DmiUF7jBtlv1UKmYFVu8rr1KXnzWmxgcwiAbSjX7eS07q6w5PF4FBZoDpNNqfArZefAWNEhaCSDzCTZ79I8mrFZyKKIfDPb5LppoP4vnOquLxOO4kkT1iPcCLl2Q2RsOGmHa8J8YmDEM%2Fyh93tym4k6p7KlIP3XLf8MgoNVCwnNwAxkIomxceXobcoDB858tuGbnZENGoQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:56:15,422 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '359'}
2023-10-04 16:56:15,423 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:57:45,373 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/wandb-metadata.json?Expires=1696495971&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=tLaEylTIQpB7KVF9D70OVIhTi8rAna5RRNsv%2B39iuLCzjmZUtEznhvwhIarunfBlV8aP9ockJDf3pceuEnvJlThK6bljxy9AKM4uVXbp2ZCljl2FGCrQFi5uAhd9%2BRvVrceUfQkGtK%2B1%2BZGLW8x%2FuNsL5OKpg6rgsj%2FsbD%2BpMg8cI7wF%2FpEmrhYyohKy9XC1Nl3DpQenXe27HAQVLxbHG87SPJNcU98vE5EWqAPHSvM2FJJ59IbJ3gquWE%2FQpIenBxMfL4berqCyfI3zYvvW%2B%2BjA576JZxzYX4wE3y6TSoN45BHUXdNGF829cdiqOQ00zihOYHGYsv5tsCyO9xYk6A%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:57:45,374 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '830'}
2023-10-04 16:57:45,374 ERROR   Thread-11 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:58:17,754 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/output.log?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=abdFuryOpbdwyc2kVfnen1GFQDp7KadJ%2BBpIMnPGpT0Qll9ZTa4LzymQFUfeZ4yN4Kfswa8WXaMGHj%2BRCjH6xb1PKQNGSZTKKGRoPQkGclXhFSbXsO64fIf5I2oduwjBU1heRpBz2mvCy3qSLgBiTFMCIT%2BFdCnzufuDbQg4twHKKOdAVH1BmZ4HBraumuBkEm76KPTWj0ZL6huhiWsu6oLlZSgsR%2FTVXtGFy9tLnrbIpibC7va2zpti7Mfpwygzn%2FlT1K5u%2F9qSZjecPxYiARrB1iqD9o3Xh4YWK7uAfbcpc7QRp1bnChngHl%2BWYBh5Ed%2BSKIuca%2FS3Nl%2Bj0s0ofQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:58:17,754 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1053'}
2023-10-04 16:58:17,754 ERROR   Thread-15 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:58:27,786 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/config.yaml?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Jnq322JRnliPFcjtgHJ5FJ5W7O1YmRQsJqW1Wv9Hunk7gjKpnz1L%2FBU6yIiVYFp5MFImlKY0lkCxD1BILzYm4%2Fo0ltBwYRrEFzPUD4zjkLq5%2BVAtw6c4TK7AtgFupRNVruJ3ws1luH9EoAZzU%2B%2F5Uxm3oZ22aKHc9vamXUlVxYSRudrNHcgXfOneIdWt0blUQclYg1Wlll%2Ff5g8Nr2NSJ35%2BjpDgv1Vvs8zS7SZOb%2Fwg0t8qyl3FSMojOLuSUpgYHKL0z36spehYND9P0Jr6XEy9MLLY8upmaKLUxC99%2Bm7ehYPzgIc5LZbfYGEtuwqi4cDBi0OYppO3aDtzd83g9A%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:58:27,786 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2405'}
2023-10-04 16:58:27,786 ERROR   Thread-14 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:58:44,912 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/wandb-summary.json?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IKd2QT561gAHYhANHPmgx0MlRG%2FmSVoskmxBB71nAtzCdVM2Pa1NhftkJFT0maBTvl%2BFm3hWenXhoBUq4LVE8nOFsSB17ekwUrhj1M9SzkmSvAH28WQXKL%2FbuxgHLu%2B9xT0dO5myycK3DmiUF7jBtlv1UKmYFVu8rr1KXnzWmxgcwiAbSjX7eS07q6w5PF4FBZoDpNNqfArZefAWNEhaCSDzCTZ79I8mrFZyKKIfDPb5LppoP4vnOquLxOO4kkT1iPcCLl2Q2RsOGmHa8J8YmDEM%2Fyh93tym4k6p7KlIP3XLf8MgoNVCwnNwAxkIomxceXobcoDB858tuGbnZENGoQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:58:44,912 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '359'}
2023-10-04 16:58:44,912 ERROR   Thread-12 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
2023-10-04 16:58:49,181 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1694] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/youyx/PMCE/3s8t6wta/requirements.txt?Expires=1696496026&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Q4xKRWSk7uGEVAmog%2BNCxXLarlYLyvYWpP9XlF6d95eDlcix63PZPWgCu7IN2kW6WFD8D0R%2BW6P%2FgHKf6rBW1zJpWzjuw1xYUFAR%2FxuZ07lwQEDIyMLUh5wUn2703wf%2BlhRnOkox9%2FRXlLlnuvIz8p5bFoqfE3ms%2BjiUt4DIimA2F5OGbroUuIYCxKnVvlSfAgZC0CB8i7sJdDMO3PahbifD9rQ4HG52xsHKaODJIM8EPCTXYWvE%2FzrQW%2BMCvkYiwUsjE999%2FO9qNP4zdoGxfssfDgnzuiXmR8OGeKdOkNkDNjjZD8kpWOgbzpe6uvNBDbjBlnK1uwWbmFJqZ9crRw%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2023-10-04 16:58:49,181 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1696] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2067'}
2023-10-04 16:58:49,181 ERROR   Thread-13 :15492 [internal_api.py:upload_file():1698] upload_file response body: 
